kernel/init.c

```diff
@@ -284,12 +293,16 @@ static void bg_thread_main(void *unused1, void *unused2, void *unused3)
 	_main_thread->base.user_options &= ~K_ESSENTIAL;
 }
 
+/* LCOV_EXCL_START */
```

Comment:
Are you sure that these comments don't introduce bugs?

Reply:
I'm not sure I understand, can you elaborate?

Reply:
Sorry, I was being facetious.